generic/tests/cfg/iometer_windows.cfg

```diff
@@ -53,3 +52,24 @@
                     cpu_family = "0xf"
                 - msi_off:
                     cpu_family = "0xe"
+        - change_vm:
+            run_timeout = 1000
+            variants:
+                - shutdown_vm:
+                    shutdown_vm = yes
+                    command_qmp = system_powerdown
+                    command_shell = shutdown -t 0 -s
+                - reboot_vm:
+                    reboot_vm = yes
+                    command_qmp = system_reset
+                    command_shell = shutdown -t 0 -r
```

Comment:
Hi yhong, I think better define another case for system_reset, like:
variants:
    - with_shutdown: 
    - with_reboot: 
    - with_system_reset: 
 
system_reset is not same as reboot, it's a force action from qemu, but reboot(shutdown -r -t 0) is a smoothly action from guest.
And:
with_shutdown only use shell method is ok, no need qmp method, qmp method is same with shell.
with_reboot only include shell method is ok, no need qmp method, qmp method is same with shell. 
with_system_reset use qmp method is ok.